site/composer.json

```diff
@@ -29,7 +29,9 @@
     "cboden/ratchet": "0.4.3",
     "textalk/websocket": "1.5.3",
     "egulias/email-validator": "3.1.1",
-    "maennchen/zipstream-php": "^2.1"
+    "maennchen/zipstream-php": "^2.1",
+    "league/commonmark": "^1.6",
+    "spatie/commonmark-highlighter": "^2.1"
```

Comment:
Use specific versions here. As Submitty isn't a library, using ranges doesn't get us anything over specific values which dependabot handles bumping.

To resolve the merge conflict though, I would recommend doing:
```bash
git checkout master
git pull master
git checkout commonmark
git add .
git commit # commit merge
wget https://raw.githubusercontent.com/Submitty/Submitty/master/site/composer.lock -O composer.lock
wget https://raw.githubusercontent.com/Submitty/Submitty/master/site/composer.json -O composer.json
cd site
composer require "league/commonmark:1.6.5"
composer require "spatie/commonmark-highlighter:2.1.1"
```
(typing this free-hand, probably some error in wget and composer usage as I can never fully remember the former's flags and the latter specifying specific version using `:` or `@`.

Reply:
Additionally, can we remove the `erusev/parsedown` dependency?

Reply:
Thanks for the help on the merge conflict. I just removed the parsedown dependency.